### DISCUSSION TITLE
Fixed BDD100K model with state_dict that matches with the config file

### DISCUSSION THREAD

**Discussion by mehta-raghav** [ethvis/qd-track#134] [Show and tell]
Here's the link to the compatible model file: https://drive.google.com/file/d/118ORtpJbVWSRf6f9CjLx4hoZ5wo2X6TY/view?usp=sharing

**Comment by Faheem-Yousafzai** [comment]
Hi Raghav,
I hope you will be fine. 

Can you share where you get the above model file? From above link, I extracted the data.pkl file and used it as checkpoint, but unfortunately it was not successful. 

Regards,
Faheem

**Reply by Raspberry-beans** [reply]
> Hi Raghav, I hope you will be fine.
> 
> Can you share where you get the above model file? From above link, I extracted the data.pkl file and used it as checkpoint, but unfortunately it was not successful.
> 
> Regards, Faheem

Resolved, thanks for sharing the file
